modules/serverless-rn-1-16-0.adoc

```diff
@@ -12,13 +12,19 @@
 * The `kn func` CLI plug-in now uses `func` 0.16.0.
 * The `kn func emit` command has been added to the functions `kn` plug-in. You can use this command to send events to test locally deployed functions.
 
-[id="fixed-issues-1-16-0_{context}"]
-== Fixed issues
-
 [id="known-issues-1-16-0_{context}"]
 == Known issues
 
-// Add a note about the following to the mTLS docs when they're ready to merge
+* The AMQ Streams Operator might prevent the installation or upgrade of the {ServerlessOperatorName}. If this happens, the following error is thrown by Operator Lifecycle Manager (OLM):
++
+[source,terminal]
+----
+WARNING: found multiple channel heads: [amqstreams.v1.7.2 amqstreams.v1.6.2], please check the `replaces`/`skipRange` fields of the operator bundles.
+----
++
+Uninstalling the AMQ Streams Operator temporarily fixes this issue.
```

Comment:
Hey @abrennan89 you might also want to say that once serverless has been upgraded, you can reinstall AMQ Streams?

Reply:
Isn't that covered by the word "temporarily"? We could also say it more explicitly.

Reply:
Yeah, I'll maybe remove "temporarily" and just say you can fix it by removing AMQ Streams but you can reinstall it after